_data/other.yml

```
@@ -567,6 +567,14 @@ websites:
       exceptions:
           text: "Two factor authentication unavailable for 'End User' access, which is read-only."
 
+    - name: Weclapp 
+      url: https://weclapp.com/
+      img: weclapp.png
+      tfa: Yes
+      software: Yes
+      hardware: Yes
+      doc: https://www.weclapp.com/de/blog/2-faktor-authentifizierung/
```

```suggestion
      doc: https://doc.weclapp.com/knowledgebase/zwei-wege-authentifizierung-2fa/
```

Change to documentation link as per @ertanoe 's request.